astronomer/providers/google/cloud/sensors/gcs.py

```diff
@@ -55,7 +66,7 @@ def __init__(
         self.delegate_to = delegate_to
         self.impersonation_chain = impersonation_chain
 
-    def execute(self, context):
+    def execute(self, context: "Context"):
```

Comment:
Return type is missing for `execute()`.

Reply:
This is already existing will be fixed as part of mypy-strict branch